fuzzers/baby/baby_fuzzer/src/main.rs

```diff
@@ -14,7 +14,7 @@ use libafl::{
     fuzzer::{Fuzzer, StdFuzzer},
     generators::RandPrintablesGenerator,
     inputs::{BytesInput, HasTargetBytes},
-    mutators::scheduled::{havoc_mutations, StdScheduledMutator},
+    mutators::{havoc_mutations::havoc_mutations, scheduled::StdScheduledMutator},
```

Comment:
Maybe we should re-export the og `havoc_mutations` to not break _all_ code out there, what do you think?

Reply:
Hmm. That's a fair point. For code we control, I'd much rather have the import from the source (`mutators::scheduled` has nothing to do with havoc_mutations anymore).

Unfortunately, deprecating re-exports is not supported yet (see [issue](https://github.com/rust-lang/rust/issues/30827)). So I guess we're stuck with just re-exporting them and add a comment telling people to do the other thing and explaining why the re-export is there. Maybe link the issue as well.

Opinions?